Documentation/project-docs/library-servicing.md

```diff
@@ -0,0 +1,33 @@
+# How to service a library in CoreFx
+
+This document provides the steps necessary after modifying a CoreFx library in a servicing branch (where "servicing branch" refers to any branch whose name begins with `release/`).
+
+## Check for existence of a .pkgproj
+
+Most CoreFx libraries are not packaged by default. Some libraries have their output packaged in `Microsoft.Private.CoreFx.NetCoreApp`, which is always built, while other libraries have their own specific packages, which are only built on-demand. Your first step is to determine whether or not your library has its own package. To do this, go into the root folder for the library you've made changes to. If there is a `pkg` folder there (which should have a `.pkgproj` file inside of it), your library does have its own package. If there is no `pkg` folder there, the library is built as part of `Microsoft.Private.CoreFx.NetCoreApp` and shipped as part of `Microsoft.NetCore.App`. If this is the case, there is nothing that needs to be done.
```

Comment:
> If there is no `pkg` folder there, the library is built as part of `Microsoft.Private.CoreFx.NetCoreApp` and shipped as part of `Microsoft.NetCore.App`

I'm not sure if we can make this claim. I believe there are still a few libraries that are not part of netcoreapp and don't yet have a pkg folder. Even though this is almost always true, I'm just not sure if we should claim that this is always the case.

Reply:
By convention there should be pgk\*.pkgproj but your right there maybe some exceptions to that rule but I don't think we need to call that out here. We should instead try and push any exceptions to follow this convention. 

It is also worth pointing out the IsNETCoreApp=true property (https://github.com/dotnet/corefx/blob/master/src/Microsoft.CSharp/Directory.Build.props#L7) can be used as an indication that you are mart of Microsoft.Private.CoreFx.NetCoreApp and transtitively part of Microsoft.NETCore.App. However that doesn't preclude you from still having your own package.